app/views/creators/_creator.html.erb

```diff
@@ -4,6 +4,7 @@
       <h5 class="card-title"><%= creator.name %></h5>
       <p class="card-text"><%= pluralize creator.models.count, "model" %></p>
       <%= link_to "Details", creator, {class: "btn btn-primary"} %>
+      <%= link_to "Models", models_path(creator: creator.id), {class: "btn btn-primary"} %>
```

Comment:
This is a bit confusing now, as both buttons go to lists of the same models, although one is a filtered `/models` list and one is using creator#show. I'd be happy to have just the link to the new unified list, though I'm not sure where the creator defaults (links, etc) would live - perhaps it could be added into the sidebar of that main list if the creator filter is defined?

Reply:
It was my thought to drop the model list from creator#show and expand creator a bit so you have have more of a full "landing page" for a creator, with links / notes / statistics, with a link to open the models.

Reply:
Yeah, that sounds sensible. Big enough to do as a followup PR instead.